src/main/java/com/blackducksoftware/ohcount4j/FileFinder.java

```diff
@@ -27,34 +28,59 @@
 import org.apache.commons.io.filefilter.IOFileFilter;
 
 public class FileFinder extends DirectoryWalker<File> {
```

Comment:
@PeterDP There is lot of code formating changes, can we include eclipse code formatter so that people can use and make sure we keep code formating standards.